lib/Service/ApiService.php

```diff
@@ -47,12 +47,18 @@ class ApiService {
 	protected $sessionService;
 	protected $documentService;
 	protected $logger;
+	private ImageService $imageService;
```

Comment:
That breaks on PHP7.3 :)

```suggestion
	protected $imageService;
```

Reply:
Right, I though Php 7.3 support had been dropped in NC 24 but it's actually not.